pkgdb2client/__init__.py

```diff
@@ -41,6 +41,9 @@ def emit(self, record):
 
 __version__ = pkg_resources.get_distribution('packagedb-cli').version
 PKGDB_URL = r'https://admin.fedoraproject.org/pkgdb/'
+FAS_URL = r'https://admin.fedoraproject.org/accounts'
+BZ_URL = r'https://bugzilla.redhat.com/xmlrpc.cgi'
+KOJI_HUB = r'http://koji.fedoraproject.org/kojihub'
```

Comment:
This would be a nice idea if there is a mean for fedora.client to receive an info for loading such default URL set from an alternative config file.

For example, it could be possible to have an environment value such as FEDORA_CLIENT_NAMESPACE=staging
, then load the default staging URL...